locales/en-US/messages.properties

```diff
@@ -166,6 +166,17 @@ homeworkExcuseGeneratorRemixTitle=Remix the {{ homeworkExcuseGenerator }} projec
 homeworkExcuseGeneratorTeachingKitTitle=Read the {{ homeworkExcuseGenerator }} teaching kit
 mySixWordSummerRemixTitle=Remix the {{ mySixWordSummer }} project
 mySixWordSummerTeachingKitTitle=Read the {{ mySixWordSummer }} teaching kit
+lessonPlan=Lesson Plan
+projectBy=By
+remixGalleryTitle=Remix a project to get started...
+projectSingular=project
```

Comment:
Let’s use something like `projectSingular={{ n }} project`, so that the number can be moved around.
Ideally, we should use proper plural form, since singular/plural won’t work for most languages:
`project[one]={{ n }} project`
`project[two]={{ n }} projects`
`project[few]={{ n }} projects`
`project[many]={{ n }} projects`

But I’m not sure what plural form support you have with this gettext implem (is this node-gettext?)